manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -1,6 +1,9 @@
 meta:
   environment: ~
 
+  api_domain: (( concat "api." properties.domain ))
+  apps_domain: (( concat "apps." properties.domain ))
```

Comment:
The story was to test environment being "basically sound" and fix small issues found. So I have fixed this as I consider it quite important that app and system domains are different.